### PR TITLE
Add a note about grpc-compression-type.

### PR DESCRIPTION
**Pull request by lilyjazz**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tidb/issues/25419

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by lilyjazz** [comment]
tech review @seiya-annie

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
 
 + gRPC 消息的压缩算法，取值：none， deflate， gzip。
 + 默认值：none
++ 在取值为 `gzip` 时，部分 TiDB Dashboard 计算无法完成，调整回默认值 `none` 可正常显示。[#25419](https://github.com/pingcap/tidb/issues/25419)
```
Comment (seiya-annie) [inline tikv-configuration-file.md:38]: 是否可以加成一个notes，或者特殊说明，因为后面的版本 issue fix了就没这个问题了。这些写感觉是一个长期方案一样。

Comment (lilyjazz) [reply tikv-configuration-file.md]: @seiya-annie 改了～

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -36,6 +36,10 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
 + gRPC 消息的压缩算法，取值：none， deflate， gzip。
 + 默认值：none
 
+> **注意：**
+>
+> - 在取值为 `gzip` 时，存在部分 TiDB Dashboard 计算无法完成的问题，调整回默认值 `none` 可正常显示。[#25419](https://github.com/pingcap/tidb/issues/25419)
```
Comment (TomShawn) [inline tikv-configuration-file.md:41]:
```suggestion
> 取值为 `gzip` 时，部分 TiDB Dashboard 可能无法完成对应的压缩运算，会显示异常。调整回默认值 `none` 后，TiDB Dashboard 可正常显示。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Review by seiya-annie — COMMENTED** [review]
LGTM

**Pull request merged**